### PR TITLE
Task/54652 send email when exam dont start in 8 hours (delayed job)

### PR DESCRIPTION
**Pull request by NeverSmileK57CLC**

### CODE REVIEW

File: bin/delayed_job [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env ruby
+
+require File.expand_path(File.join(File.dirname(__FILE__), '..', 'config', 'environment'))
```
Comment (hellovietnam93) [inline bin/delayed_job]: using ""

File: config/initializers/config.rb [hunk 2]
```diff
@@ -1,3 +1,8 @@
 Config.setup do |config|
   config.const_name = "Settings"
 end
+if Rails.env.production?
+  DelayedJobWeb.use Rack::Auth::Basic do |username, password|
+    username == "username" && password == "password"
```
Comment (hellovietnam93) [inline config/initializers/config.rb]: "username" should be add to environment variable

Comment (NeverSmileK57CLC) [inline config/initializers/config.rb]: e định làm mà lại sợ quá file :3

**Comment by hellovietnam93** [comment]
OK

File: bin/delayed_job [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env ruby
+
+require File.expand_path(File.join(File.dirname(__FILE__), "..", "config", "environment"))
+require 'delayed/command'
```
Comment (dieunb) [inline bin/delayed_job]: double quote

**Pull request merged**